CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -481,17 +483,168 @@ public async Task MoveGameLocation()
         public async ValueTask<bool> UninstallGame()
         {
             string GameFolder = ConverterTool.NormalizePath(_gamePath);
-
             switch (await Dialog_UninstallGame(_parentUI, GameFolder, _gameVersionManager.GamePreset.ZoneFullname))
             {
                 case ContentDialogResult.Primary:
                     try
                     {
-                        Directory.Delete(GameFolder, true);
+                        string gameDataFolderName = ""; // ExecutableName_Data
+                        string[] filesToDelete = { }; // Put whatever files it needs to delete in the root of GameFolder
```

Comment:
I think putting them directly in the `ConfigV2Store` would be better, but unsure if it's required.

Maybe the end-goal could be to standardize the files to delete, since there are some similarities between different `GameType`s.

Reply:
putting it in ConfigStore would be ideal, but it will clog up metadata for something that is mostly static so I don't see the need for this

Reply:
Fair point, I think that we can leave it as it is for the time being then.